isis/src/control/objs/BundleUtilities/AbstractBundleObservation.h

```diff
@@ -73,7 +72,7 @@ namespace Isis {
       virtual QStringList parameterList();
       virtual QStringList imageNames();
 
-    private:
+    protected:
```

Comment:
The members below here are duplicated in the BundleObservation class. They probably need to go away.

Reply:
Agreed. Making the values protected in the parent and removing from the child classes works as a solution and I merged this, but am still potentially interested in investigating if it's possible to handle weights/corrections/apriori/adjustedsigmas at the parent level if we were to modify how these are applied to be be more generalized.